test/runtests.jl

```diff
@@ -1,7 +1,11 @@
+using LightXML
+using Compat
+@static VERSION < v"0.7.0-DEV" ? (using Base.Test) : (using Test)
```

Comment:
use Compat for this

Reply:
Again I tried, but it got errors on v0.6.2 still.
It should be `@compat using Test` ?

Reply:
I believe all you need to do is:
```julia
using Compat.Test
```

Reply:
OK! Thanks!